src/app/components/add-card/add-card.component.ts

```diff
@@ -81,6 +81,16 @@ export class AddCardComponent implements OnDestroy {
     this.newTask = '';
   }
 
+  onTaskButtonClick(event: {id: number, dataJob: string}): void {
```

Comment:
Could we not destructure `id` and `dataJob` instead of `event`?

Reply:
yes we can. ` onTaskButtonClick({id, dataJob}: {id: number, dataJob: string})`. updated!